src/org/usfirst/frc/team2473/robot/Robot.java

```diff
@@ -60,13 +54,21 @@ public void robotInit() {
 		SmartDashboard.putData("Power", powerChooser);
 		SmartDashboard.putData("Distance", distanceChooser);
 		
+		UsbCamera camera0 = CameraServer.getInstance().startAutomaticCapture("Cube View", 0);
```

Comment:
We should name these cameras adequately for clarity.

```suggestion
		UsbCamera cubeViewCamera = CameraServer.getInstance().startAutomaticCapture("Cube View", 0);
```

Reply:
The same goes for the other camera